### DISCUSSION TITLE
Nesting `@media print` inside `@layer`

### DISCUSSION THREAD

**Discussion by brandonpittman** [tailwindlabs/tailwindcss#9967] [Help]
I've got some global print styles. I thought putting them alongside my base styles in an `@layer base` block would be the right way to go. That results in the `@media` block being stripped out. If I move the `@media` block outside the layer, all is well.

Can `@page` and `@media` blocks not be nested inside an `@layer`?

**Comment by wongjn** [comment]
Yes they should be fine in `@layer`, I've tested a `@media print` in a [Tailwind Play](https://play.tailwindcss.com/8c4d7zyDNG?file=css) and it is preserved as expected. If you are using a class name, ensure it appears in a file covered by the `content` file globs defined in the Tailwind configuration. Otherwise, it could be some tooling running after Tailwind that is stripping the media query out.
